api/client/task/print.go

```diff
@@ -41,7 +41,7 @@ func (t tasksBySlot) Less(i, j int) bool {
 }
 
 // Print task information in a table format
-func Print(dockerCli *client.DockerCli, ctx context.Context, tasks []swarm.Task, resolver *idresolver.IDResolver) error {
+func Print(dockerCli *client.DockerCli, ctx context.Context, tasks []swarm.Task, resolver *idresolver.IDResolver, noTrunc bool) error {
```

Comment:
So i kinda had the same thought but `docker ps` passes it around as `noTrunc` so i kept it.